docs/release/3.5/release-3-5-0.md

```diff
@@ -0,0 +1,156 @@
+    [policy](https://opensciencegrid.org/technology/policy/release-series/):
+    regular support will end in **February 2020** and critical bug/security support will end in **November 2020**.
+
+### Package updates ###
+
+In addition to the packages that were carried over from [OSG 3.4.33](/release/3.4/release-3-4-33#),
+this release contains the following package updates:
+
+-   HTCondor 8.8.4: The current HTCondor [stable release](https://htcondor.readthedocs.io/en/v8_8_4/version-history/stable-release-series-88.html).
+    See the [manual upgrade instructions](/release/release_series#updating-to-htcondor-8.8.x) before
+    updating to this version.
+-   CVMFS 2.6.2: A [bug fix release](https://cvmfs.readthedocs.io/en/2.6/cpt-releasenotes.html).
+    Note the upgrade recommendations from the developers:
+
+    > As with previous releases, upgrading clients should be seamless just by installing the new package from the
+    > repository.
+    > As usual, we recommend to update only a few worker nodes first and gradually ramp up once the new version proves
+    > to work correctly.
+    > Please take special care when upgrading a cvmfs client in NFS mode.
+
+    > For Stratum 1 servers, there should be no running snapshots during the upgrade.
+    > For publisher and gateway nodes, all transactions must be closed and no active leases must be present before
+    > upgrading.
+
+    See the known issue with this version [below](#known-issues).
+
+-   XCache 1.1.1: This release includes packages for ATLAS and CMS XCaches as well as Stash Origin HTTP/S support.
+-   OSG Configure 3.0.0: A [major version release](https://github.com/opensciencegrid/osg-configure/releases/tag/v3.0.0),
+    including changes from the OSG Configure 2.4 series and dropping some deprecated features.
+    See the [manual upgrade instructions](/release/release_series#updating-to-osg-configure-3) before updating to this
+    version.
+-   OSG XRootD 3.5: A meta-package including common configuration across [standalone](/data/xrootd/install-standalone),
+    [storage element](/data/xrootd/install-storage-element), and [caching](/data/stashcache/overview) installations of
+    XRootD.
+-   XRootD LCMAPS 1.7.4: includes default authorization configuration in `/etc/xrootd/config.d/40-xrootd-lcmaps.cfg`.
+    To use the default configuration, uncomment the `# set EnableLcmaps = 1` line in `/etc/xrootd/config.d/10-xrootd-lcmaps.cfg`.
+-   XRootD HDFS 2.1.6: includes default configuration in `/etc/xrootd/40-xrootd-hdfs.cfg`.
+-   MyProxy 6.2.4: Remove usage statistics collection support
+-   [CCTools 7.0.14](http://ccl.cse.nd.edu/software/): Bug fix release
+-   OSG System Profiler 1.4.3: Remove collection of obsolete information
+    See the known issue with this version [below](#known-issues).
+-   Upcoming repository:
+    - HTCondor 8.9.2: The current HTCondor [development release](https://htcondor.readthedocs.io/en/v8_9_2/version-history/development-release-series-89.html)
```

Comment:
@timtheisen could you fill in some highlights of the 8.9 series here?

Reply:
Here are the highlight from our release announcements:
- All HTCondor daemons run under a condor_master share a security session
- Scheduler Universe jobs are prioritized by job priority
- An efficient curl plugin that supports uploads and authentication tokens
- HTCondor automatically supports GPU jobs in Docker and Singularity
- File transfer times are now recorded in the user job log and the job ad
- The HTTP/HTTPS file transfer plugin will timeout and retry transfers
- A new multi-file box.com file transfer plugin to download files
- Configuration options for job-log time-stamps (UTC, ISO 8601, sub-second)
- Several improvements to SSL authentication
- New TOKEN authentication method enables fine-grained authorization control

Reply:
The curl plugin is the HTTP/S plugin